script/server

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# script/server: Launch the application and any extra required processes
+#                locally.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if ps x |grep -v grep |grep -c redis-server; then
```

Comment:
@tarebyte This seems to work as needed. I did npm start with redis running and without it running and the result was the same. I think it's good to go, but I'll wait for your opinion